ui_tests/playwright_tests.cljs

```diff
@@ -64,15 +64,21 @@
                                          (not (str/ends-with?
                                                (.-url (.location msg)) "favicon.ico")))
                                 (swap! console-errors conj {:msg msg :notebook (.url page)}))))
+                     _ (.on page "pageerror"
```

Comment:
Do these changes fix #485?

Reply:
It should be an improvement in general, but alas it didn't fix that error as the event wasn't triggered.  It can't hurt though.

See:
https://stackoverflow.com/questions/73448785/how-to-detect-uncaught-in-promise-typeerror-in-playwright/73459509#73459509